wordssite/templates/dictionary/word_confirm_delete.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+{% url 'dictionary:all' as list_all %}
+
+<div class="container d-flex justify-content-center mt-5" style="width: 50%;">
+    <div class="col mt-5 text-center">
+        <h1 class="mb-5">Delete Word</h1>
+
+        <p class="h5">Are you sure you want to delete the Word:</p>
+        <h3 class="my-5">{{word.word}}</h3>
+        <form action="" method="POST">
+            {% csrf_token %}
+            {% comment %} <input type="submit" value="Confirm">
+            <input type="submit" value="Cancel" onclick="window.location='{% url 'dictionary:all' %}'; return false;"> {% endcomment %}
```

Comment:
what does this window.location do?

Reply:
When a user is clicking on the Cancel/Confirm it brings the user back to his dictoanry words

Reply:
https://stackoverflow.com/questions/1667416/window-location-href-vs-clicking-on-an-anchor -deleted